web/static.go

```diff
@@ -81,7 +81,11 @@ func staticFilesHandler(handler http.Handler) http.Handler {
 		//wrap our ResponseWriter with our no-cache 404-handler
 		w = &notFoundNoCacheResponseWriter{ResponseWriter: w}
 
-		w.Header().Set("Cache-Control", "max-age=31556926, public")
+		if path.Base(r.URL.Path) == "remote_entry.js" {
+			w.Header().Set("Cache-Control", "no-cache, max-age=0")
```

Comment:
This should actually be `no-store, max-age=0`. No-cache means the response can be cached, but must be revalidated: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control

Reply:
Oh, okay. Maybe "don't cache remote_entry.js" isn't what I want to do then because the web app HTML seems to be cached the way I want, but it uses `no-cache, max-age=31556926, public` as above. Basically, the browser/CDN can store it, but it should always make sure to double check that it hasn't changed which seems to work on community by looking at the network requests made for HTML files

![Screen Shot 2022-12-08 at 11 04 48 AM](https://user-images.githubusercontent.com/3277310/206498237-0e26e61d-2ebf-469a-8fe3-6fe5902e97c3.png)

I'm also seeing there that the `cf-cache-status` header is being set to `DYNAMIC` for the HTML files instead of `HIT`/`MISS` as it is for the other JS files

Reply:
I changed this to match how the `root.html` file is cached. Let me know if that sounds like it should work